Source/Engine/Platform/Base/PlatformBase.h

```diff
@@ -316,6 +316,25 @@ DECLARE_SCRIPTING_TYPE_MINIMAL(PlatformBase);
     /// <param name="ptr">A pointer to the memory block to deallocate.</param>
     static void Free(void* ptr) = delete;
 
+    /// <summary>
+    /// Returns the OS's default page size that can be used with AllocatePages.
+    /// </summary>
+    static uint64 GetDefaultPageSize() = delete;
+
+    /// <summary>
+    /// Allocates pages memory block.
+    /// </summary>
+    /// <param name="numPages">The number of pages to allocate.</param>
+    /// <param name="pageSize">The size of single page. Use GetDefaultPageSize() or provide compatible, custom size.</param>
```

Comment:
Update comment to use `Platform::GetCPUInfo().PageSize` as default.

Reply:
Updated.